codegen/generate-manager-classes.tsx

```diff
@@ -0,0 +1,86 @@
+import * as path from 'path';
+import * as ts from 'typescript';
+import { createClassForOperations } from './create-class-for-operations';
+import { generateInterfacesForSchema } from './generate-interfaces-for-schema';
+import { OpenAPI } from './openapi';
+import * as tsx from './tsx';
+import {
+	ExportAssignment,
+	Identifier,
+	ImportClause,
+	ImportDeclaration,
+	StringLiteral,
+} from './tsx';
+import { writeNodesToFile } from './write-nodes-to-file';
+tsx;
+
+const SCHEMAS_RELATIVE_PATH = '../src/schemas';
+
+export async function generateManagerClasses(
```

Comment:
It's not clear why changing `generateManagerClass` into `generateManagerClasses` was necessary. In general, the original singular version of the function is arguably more generic/easier to test etc. This is just the previous prototype for codegen that we're not actively maintaining, so no strong opinions here though.